### ISSUE TITLE
Char dump missing history

### ISSUE THREAD

**Issue by angband-import** [angband/angband#1746]
**Reported by SaThaRiel on 29 Nov 2010 17:18 UTC**
When doing a char dump in the current dev version (commit 8ceb01d9ea692f6953def223ba92bcf5230635be) the char history is empty...just some lines and then the options are displayed:

```[...]
f) 2 Wooden Torches (5000 turns)
     Radius 2 light, reduced when running out of fuel.


============================================================
                   CHAR.
|   TURN  | DEPTH |LEVEL| EVENT
============================================================



  [Options]

Maximize effect of race/class bonuses        : yes (adult_maximize)
Randomize some of the artifacts (beta)       : no  (adult_randarts)
[...]
```
Playing on ArchLinux (Kernel 2.6.36), compiled with gcc 4.5.1 20101125 (prerelease).
Bug was reproducible with an existing and a new character.

**Comment by angband-import** [comment]
**Comment by magnate on 29 Nov 2010 18:12 UTC**
Confirmed. Not sure how this happened.

**Comment by angband-import** [comment]
**Comment by magnate on 29 Nov 2010 18:44 UTC**
Fixed in http://bit.ly/eXwNae

**Comment by angband-import** [comment]
**Modified by takkaria on 29 Nov 2010 22:19 UTC**

**Comment by angband-import** [comment]
**Comment by magnate on 29 Nov 2010 22:55 UTC**
Fixed in adf9766c
